src/actors/hcd/neutral_fueling_actor.jl

```diff
@@ -0,0 +1,412 @@
+#= ====== =#
+#  PELLET  #
+#= ====== =#
+Base.@kwdef mutable struct FUSEparameters__ActorNeutralFueling{T<:Real} <: ParametersActor{T}
+    _parent::WeakRef = WeakRef(nothing)
+    _name::Symbol = :not_set
+    _time::Float64 = NaN
+    taupin::Entry{Float64} = Entry{Float64}("s", "Particle confinement time"; default=1.0)
+    twall::Entry{Float64} = Entry{Float64}("eV", "Wall temperature"; default=10.0)
+    vwall::Entry{Float64} = Entry{Float64}("m/s", "Wall particle velocity"; default=100.0)
+    model::Switch{Symbol} = Switch{Symbol}([:neucg, :none], "-", "Neutral gas fueling model"; default=:neucg)
```

Comment:
I think this is a good moment to talk about model = :none

This also happens elsewhere (ActorCoreTransport for example) 

What I think should happen is that these actors shouldnt' be executed and instead use ActorNoOperation in the compound ones (kind of like how we do this for the pedestal)

what do you think @orso82 ?